recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/gles-module-egl-headers_1.7.bb

```diff
@@ -1,4 +1,15 @@
-S = "${WORKDIR}/git"
+require gles-user-module.inc
```

Comment:
You can move most of this file to `gles-module-egl-headers.inc` to minimize duplication.

Also, it is not clear why `gles-module-egl-headers` require `gles-user-module.inc`.